index.html

```diff
@@ -6,9 +6,224 @@
   <title>Get Aggie</title>
   <meta name="description" content="Get Aggie">
   <meta name="author" content="Technology and International Development Lab">
+
+  <link rel="stylesheet" href="vendor/bootstrap/css/bootstrap.min.css" />
+  <link rel="stylesheet" href="vendor/bootstrap/css/bootstrap-theme.min.css" />
+  <link rel="stylesheet" href="vendor/lightbox-4.0.2/ekko-lightbox.min.css" />
+  <link rel="stylesheet" href="css/getaggie.css" />
 </head>
 
 <body>
-  <h1>Get Aggie</h1>
+  <nav>
+    <ul>
+      <li><a href="https://github.com/TID-Lab/aggie">Github</a></li>
+      <li><a href="https://github.com/TID-Lab/aggie">Documentation</a></li>
+    </ul>
+  </nav>
+  <header>
+    <img src="img/logo.png" alt="Aggie Logo" />
+    <h1 class="sr-only sr-only-focusable">Aggie</h1>
+    <h2>
+      Aggie is an open-source social and online media aggregation tool
+      designed specifically for monitoring events in real time.
+    </h2>
+    <button class="btn">Request a demo</button>
+  </header>
+
+  <section>
+    <h1 class="sr-only sr-only-focusable">Features</h1>
+    <div class="media row">
```

Comment:
@hooverlunch: Let's check in about which images go to what feature (which thumbnail and which larger image/gif).

I also need text for the content for features.